README.md

```diff
@@ -1063,7 +1063,7 @@ const EqStandard: Eq<User> = struct({
 
 const EqID: Eq<User> = pipe(
   N.Eq,
-  contramap((_: User) => _.id)
+  contramap((user: User) => user.id)
```

Comment:
_This might not fall into grammatical corrections, so please let me know if you don't want to change any code, and I'll remove this part._

What confused me with `_` variable naming for `User` is that `_` is commonly used for non-used variables. Is it used like that for brevity here?

I'd even suggest something like:
```suggestion
  contramap(({ id }: User) => id)
```

Reply:
It’s fine.